doc/coloring/coloring-family.rst

```diff
@@ -73,12 +73,12 @@ Column           Type        Description
 
 .. result columns start edgeColoring
 
-Returns SET OF ``(edge_id, color_id)``
+Returns SET OF ``(id, color_id)``
 
 ===============  =========== ====================================================
 Column           Type        Description
 ===============  =========== ====================================================
-**edge_id**      ``BIGINT``  Identifier of the edge.
+**id**           ``BIGINT``  Identifier of the edge.
```

Comment:
From here:
https://docs.pgrouting.org/3.3/en/pgr_edgeColoring.html
The returned column is `edge_id`, not `id`
Please also change the double star to double back tick
```
``edge_id``
```

Reply:
Ma'am did the fixes that you asked for.